news/_posts/2020-05-13-the-new-order-pages-in-prestaShop-1.7.7.md

```diff
@@ -0,0 +1,225 @@
+
+![Order's messages](/assets/images/2020/05/messages-order.png)
+
+***What is the goal***:
+
+The interface follows the standard interface of the messages of our daily devices. The differentiation between private messages only visible by the merchant and sent messages to the customer is now more distinct. This allows a better reading and avoids misunderstandings.
+
+***How it works***: 
+
+At the 5th message between the customer and the merchant a link is displayed for the pop-in. It displays the entire conversation of the order.
+
+
+
+
+- **Hook**: The redesign of this page reviews the hierarchy of information and refreshes the user interface with an important impact on the position of the hooks.
+
+***What is the goal***: 
+
+The hooks are repositioned in more strategic locations on the page and provide a better user experience.
+
+***How it works***: 
+
+Find [here](https://devdocs.prestashop.com/1.7/modules/core-updates/1.7.7/) the documentation on the hook&#39;s mapping. A new hook grouping the actions of the top of the order page has also been added to replace displayBackOfficeOrderActions which will no longer have the same layout.
+
+![Order hooks mapping](/assets/images/2020/05/order_details_hooks.jpg "Order hooks mapping")
+
+The detailed changes can be found on the following pull requests on github [16144](https://github.com/PrestaShop/PrestaShop/pull/16144) and [17734](https://github.com/PrestaShop/PrestaShop/pull/17734).
+
+A summary of the hooks impacted:
+
+  - displayAdminOrderTop (alias of displayInvoice)
+  - displayBackOfficeOrderActions
+  - displayAdminOrderSide (replace displayAdminRight)
+  - displayAdminOrderMain (replace displayAdminLeft)
+  - displayAdminOrderTabLink (remplace displayAdminOrderTabOrder et displayAdminOrderTabShip)
+  - displayAdminOrderTabContent (remplace displayAdminOrderContentOrder and displayAdminOrderContentShip)
+  - actionGetAdminOrderButtons
+
+Over the last few months, PrestaShop has been working hard to deliver its new features and fix bugs on the page. 31 bugs, including 8 majors, 16 minors and 3 trivials, have been fixed from the previous 1.7 and 1.6 versions. For more information, you can find [the way we classified the severity of the bug](https://build.prestashop.com/news/severity-classification/) and [the list of the fixed bugs](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3AOrder+milestone%3A1.7.7.0+label%3ABug+-label%3ARegression+sort%3Acreated-asc+is%3Aclosed). Overall, the order project has ensured more stability by correcting ⅔ of the major bugs of the order page.
+
+We encourage you to try out the features on the beta version and share any feedback [here](https://github.com/PrestaShop/PrestaShop/issues/new/choose). Your opinion is important to us. We are counting on you to share your feedback and new ideas with us so that we can continue to develop the Prestashop that you need.
```

Comment:
DAMNIT MATEUS
```suggestion
We encourage you to try out the features on the beta version and share any feedback [here](https://github.com/PrestaShop/PrestaShop/issues/new/choose). Your opinion is important to us. We are counting on you to share your feedback and new ideas with us so that we can continue to develop the PrestaShop that you need.
```